src/Controller/Action/Admin/GenerateExamplePdfAction.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusGiftCardPlugin\Controller\Action\Admin;
+
+use Setono\SyliusGiftCardPlugin\Factory\GiftCardFactoryInterface;
+use Setono\SyliusGiftCardPlugin\Form\Type\GiftCardConfigurationType;
+use Setono\SyliusGiftCardPlugin\Generator\GiftCardPdfGeneratorInterface;
+use Setono\SyliusGiftCardPlugin\Model\GiftCardConfigurationInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Webmozart\Assert\Assert;
+
+final class GenerateExamplePdfAction
```

Comment:
So either this action should return the PDF and we will encode it client side (with https://developer.mozilla.org/en-US/docs/Web/API/btoa) or this action should be renamed to `GenerateEncodedExamplePdfAction` I think. Wdyt?